docs/helmrelease-guide/values.md

```diff
@@ -0,0 +1,138 @@
+# Values
+
+Now that we have a good understanding of where we can get our charts from and
+what they have to offer, it is time to examine how you can supply
+[values](https://helm.sh/docs/glossary/#values-values-files-values-yaml) to
+be used with the chart when the Helm Operator makes a release.
+
+### Inlined values
+
+The most simple option to define the values for your Helm release. This is a
+YAML map as you would put in a file and supply to Helm with `-f values.yaml`,
+but inlined into the `HelmRelease` manifest. For example:
+
+```yaml
+spec:
+  values:
+    foo: value1
+    bar:
+    baz: value2
+    oof:
+    - item1
+    - item2
+```
+
+## Values from sources
```

Comment:
Can you please mention the downside of this feature is that changes made on these values take more time to be propagated (as per #151)?